test/data_handler/test_data_silo.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
```

Comment:
lets leave this fct in the top level folder with all the other tests.
If we restructure the test folder we will do so in a separate PR and for all tests

Reply:
done.